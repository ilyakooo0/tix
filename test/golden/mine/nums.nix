rec {
  f = x: y: x + y;
  a = f 1 2;
  b = f 1.1 1;
}
