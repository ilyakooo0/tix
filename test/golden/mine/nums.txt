{a = Number; b = Number; f = ∀ α. α -> α -> α;}